src/api/query.request.ts

```diff
@@ -121,3 +121,18 @@ export const lookupAccountAppLocalStates = (addr: string, appId: number) => {
     }
   }`;
 };
+
+/** LookUpSigmaDaoByAppId
+ * @param appId Application ID
+ */
+export const getSearchByAppIdReq = (appId: number) => {
+	return `query SearchByAppId {
```

Comment:
may be we can update it to `getDaoInfoByAppIdReq`

Reply:
Done